packages/react-dom/src/client/ReactDOM.js

```diff
@@ -811,7 +810,8 @@ function createRoot(container: DOMContainer, options?: RootOptions): ReactRoot {
 
 if (enableStableConcurrentModeAPIs) {
   ReactDOM.createRoot = createRoot;
-  ReactDOM.unstable_createRoot = undefined;
+} else {
```

Comment:
wont this deopt ReactDOM object once again?

Reply:
Was mistakenly pulled from an old branch.